cosmic-settings/src/pages/power/mod.rs

```diff
@@ -0,0 +1,22 @@
+use cosmic_settings_page as page;
+
+pub mod profiles;
+
+pub mod backend;
+
+#[derive(Default)]
+pub struct Page;
+
+impl page::Page<crate::pages::Message> for Page {
+    fn info(&self) -> page::Info {
+        page::Info::new("power", "battery-symbolic")
+            .title(fl!("power"))
+            .description(fl!("power", "desc"))
+    }
+}
+
+impl page::AutoBind<crate::pages::Message> for Page {
+    fn sub_pages(page: page::Insert<crate::pages::Message>) -> page::Insert<crate::pages::Message> {
+        page.sub_page::<profiles::Page>()
```

Comment:
This doesn't need to be a sub-page. It should be a section of this page. See the mockup here: https://github.com/pop-os/cosmic-settings/issues/10

Reply:
I was planning implementing sleep/screen saver settings. And i thought structure should be like general power page and sub-pages.

Power
     -- Sleep
     -- Power Profiles
 
Won't be so complicated if we put all power related settings in same page? Because we can still add things to power profiles sub page like auto switch power saver mode below 10% battery etc.